controllers/IssueController.js

```diff
@@ -459,6 +463,28 @@ const deleteIssue = async (req, res) => {
   }
 };
 
+/**
+ * Filters Issue properties from the given object
+ * @param {Document} issue Mongoose Issue Object
+ * @param {String} userId ID of the current user
+ * @returns {Document} issue object with only required properties
+ */
+const filterIssueProperties = function (issue, userId) {
+  const filteredIssue = JSON.parse(JSON.stringify(issue));
+
+  filteredIssue["isAuthor"] = filteredIssue.createdBy.toString() === userId;
+  filteredIssue["hasUpvoted"] = !!filteredIssue.upvoters.find(
+    (id) => id.toString() === userId
+  );
+
+  delete filteredIssue.__v;
+  delete filteredIssue.isDeleted;
+  delete filteredIssue.createdBy;
+  delete filteredIssue.upvoters;
```

Comment:
Why are we deleting count of upvoters to be returned back?

Reply:
My bad I didn't see that we were sending `upvotes` back in the response. I'll mark it as resolved.